lms/static/sass/course/_student-notes.scss

```diff
@@ -232,6 +237,15 @@ $divider-visual-tertiary: ($baseline/20) solid $gray-l4;
           a.reference-meta {
             @extend %shame-link-text;
           }
+
+          a.reference-meta.reference-tags:after {
+            content: ",";
+            color: $gray;
```

Comment:
using $m-gray-d2 is best here, as you noted in hipchat. 

NIT: additionally, you could move .reference-tags:after { } and .reference-tags:last-child:after {} so that it is nested within the .reference-meta styles that start on line 231 above.